src/transit_equity/census/income.py

```diff
@@ -0,0 +1,141 @@
+        label='100000_to_124999', min_income=100000, max_income=124999)
+    B19001_015E: IncomeDetails = IncomeDetails(field='B19001_015E',
+        label='125000_to_149999', min_income=125000, max_income=149999)
+    B19001_016E: IncomeDetails = IncomeDetails(field='B19001_016E',
+        label='150000_to_199999', min_income=150000, max_income=199999)
+    B19001_017E: IncomeDetails = IncomeDetails(field='B19001_017E',
+        label='200000_or_more', min_income=200000, max_income=200000)
+
+# The main motive of this dictionary is to map the column names to human readable names
+INCOME_DISTRIBUTION_COLUMNS_DICT = {
+    'B19001_001E': 'total',
+    'B19001_002E': 'less_than_10000',
+    'B19001_003E': '10000_to_14999',
+    'B19001_004E': '15000_to_19999',
+    'B19001_005E': '20000_to_24999',
+    'B19001_006E': '25000_to_29999',
+    'B19001_007E': '30000_to_34999',
+    'B19001_008E': '35000_to_39999',
+    'B19001_009E': '40000_to_44999',
+    'B19001_010E': '45000_to_49999',
+    'B19001_011E': '50000_to_59999',
+    'B19001_012E': '60000_to_74999',
+    'B19001_013E': '75000_to_99999',
+    'B19001_014E': '100000_to_124999',
+    'B19001_015E': '125000_to_149999',
+    'B19001_016E': '150000_to_199999',
+    'B19001_017E': '200000_or_more',
+}
+
+# TODO: This function can be extended to do a calculation on an entire pandas DataFrame
+# The extended function will also have better performance.
+# TODO: Create another function that returns the columns that fall within the income range
+# Such a function is more extensible as we can do more nuanced analysis, with less assumptions about the data
+def get_households_in_income_range(income_distribution_row: pd.Series, min_income: int, max_income: int) -> int:
+    '''
+    A function to get the number of households in a given income range from a census row.
+    Since the range is variable, we will have to check all the columns in the series that are present in the range.
+    If a column is partially in the range, we will consider the whole column.
+
+    Parameters:
+    ----------
+    census_row: pd.Series
+        A pandas series containing the income distribution data
+        Should contain all the columns in transit_equity.census.income.INCOME_DISTRIBUTION_COLUMNS
+    
+    min_income: int
+        Left end of the income range
+    
+    max_income: int
+        Right end of the income range
+    
+    Returns:
+    -------
+    float
+
+    Examples:
+    --------
+    Example 1:
+    >>> import pandas as pd
+    >>> from transit_equity.census.income import INCOME_DISTRIBUTION_COLUMNS
+    >>> from transit_equity.census.income import get_households_in_income_range
+    >>> income_distribution_dict = dict()
+    >>> for column in INCOME_DISTRIBUTION_COLUMNS:
+    ...     income_distribution_dict[column.value.field] = 0
+    >>> income_distribution_row = pd.Series(income_distribution_dict)
+    >>> get_households_in_income_range(income_distribution_row, 0, 0)
```

Comment:
I guess this specific example may not be the most useful since it only mentions an edge case.
Is this still fine? Any thoughts @bhazelton ?

Reply:
same suggestion here: explain what it is doing and why

Reply:
Similar update here @bhazelton